bundle.js

```diff
@@ -33,6 +33,31 @@ var accname = (function (exports) {
         return element;
     }
 
+    /*
+      util.ts contains helper functions that are used by more than one rule.
+    */
+    /**
+     * Calculates whether or not a given element is focusable.
+     * @param elem - The element whose focusability is to be calculated.
+     */
+    function isFocusable(elem) {
+        // See https://html.spec.whatwg.org/multipage/interaction.html#the-tabindex-attribute
```

Comment:
Should this be `#focusable-area`?

Reply:
The specific area that I was looking at is actually under the tabindex definition:
![image](https://user-images.githubusercontent.com/34962541/93482436-9f36ca00-f8f7-11ea-8438-ef1417cdc16f.png)